maps/tgstation.dmm

```diff
@@ -967,7 +967,7 @@
 "asE" = (/obj/machinery/atmospherics/unary/vent_pump{dir = 8; level = 2; on = 1},/turf/simulated/floor,/area/security/brig)
 "asF" = (/obj/structure/closet/secure_closet/brig{id_tag = "Cell 2"; name = "Cell 2 Locker"},/obj/machinery/camera{c_tag = "Brig Cell 2"},/turf/simulated/floor{icon_state = "red"; dir = 4},/area/security/brig)
 "asG" = (/obj/structure/cable{d1 = 1; d2 = 2; icon_state = "1-2"},/turf/simulated/floor{icon_state = "red"; dir = 8},/area/security/lobby)
-"asH" = (/obj/structure/cable{d1 = 4; d2 = 8; icon_state = "4-8"},/obj/structure/window,/turf/simulated/floor,/area/security/lobby)
```

Comment:
https://a.pomf.cat/exztlp.png  I'll be damned if I can find this mysterious wire.